irteus/irtgeo.l

```diff
@@ -69,7 +69,7 @@
      dif-pos))
   (:difference-rotation
    (coords &key (rotation-axis t))
-   "return difference in rotation of given coords, rotation-axis can take (:x, :y, :z, :xx, :yy, :zz, :xm, :ym, :zm)"
+   "return difference in rotation of given coords, rotation-axis can take (:x, :y, :z, :xy, :yz, :zx, :xx, :yy, :zz, :xm, :ym, :zm)"
```

Comment:
How about adding `:yx :zy :xz`?

Reply:
The documentation of `:difference-position` does not have `:yx :zy :xz`.
I made the documentation of `:difference-rotation` similar to `:difference-position`.
https://github.com/euslisp/jskeus/blob/89026425eeea39ea483872d6885f8decfaa05fac/irteus/irtgeo.l#L56-L58

Reply:
I see.
But `rotation-axis`, the argument of `:difference-rotation`, can take `:yx, :zy, :zx`. This is clear from your commit. https://github.com/euslisp/jskeus/pull/568/commits/c3603fefd3a7329e1d4002ac51023817f92d70a7#diff-23ea0c1757d7ef177d46886dd7b78a15R91

I think that good documentation supports other users. So how about changing the document of `difference-position`?

Reply:
It ’s true. I fixed. https://github.com/euslisp/jskeus/pull/568/commits/85c060d39f347120dc426c8cd1d37f85cd2d3fbc
Thank you!